master/internal/rm/multirm/multirm_intg_test.go

```diff
@@ -0,0 +1,685 @@
+//go:build integration
```

Comment:
does this need to be intg? or can it just be a unit test?

Reply:
yeah, it can be unit, I took out the one test that made it intg ...